src/sentry/api/event_search.py

```diff
@@ -103,7 +104,7 @@
 text_in_filter = negation? text_key sep text_in_list
 
 # standard key:val filter
-text_filter = negation? text_key sep search_value
+text_filter = negation? text_key sep operator? search_value
```

Comment:
Will we support IN filters with semver in the future?

Reply:
No, I don't think it really makes sense. Can use `IN` with just the standard release string if needed